src/others/sysmon/lv_sysmon.c

```diff
@@ -144,10 +143,25 @@ static void perf_monitor_disp_event_cb(lv_event_t * e)
             break;
     }
 }
-uint32_t lv_os_get_idle_percent(void);
 
 static void perf_update_timer_cb(lv_timer_t * t)
 {
+    static bool inited = false;
```

Comment:
Multiple instances of lvgl will cause problems, consider moving to `lv_global_t`.

Reply:
True, fixed.